RELEASES.md

```
@@ -0,0 +1,84 @@
+- **Connect Upgrade Testing:** Do a version upgrade from 2.57.0 to the latest version with a fully set up Personal ID login with opportunities and on upgrade, we should not need to re-login into Personal ID again and should see all the Connect related data as it was before the upgrade, this will need to be tested on all Connect screens.
+
+- Test manual and automatic GPS capture and verify that the location values are saved correctly inside a CC form.
+
+- Unsent forms correctly appear on the App Home Screen.
+
+- **Form Entry**
+  - Test that backing out or completing the form works correctly.
+  - Test that form entry with phone rotation is saved and submitted successfully to the server.
+  - Test incomplete form save.
+
+- **Login Screen:** A video shared [here](https://dimagi.atlassian.net/browse/CI-467) shows steps to reproduce the bug. Verify that it's fixed, and also test various app switching and login scenarios on Login Screen.
```

markdownlint-cli2 (0.21.0)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)